central/cve/fetcher/istio.go

```diff
@@ -67,7 +68,13 @@ func (m *istioCVEManager) setCVEs(cves []*storage.EmbeddedVulnerability, nvdCVEs
 	m.embeddedCVEs = cves
 }
 
-func (m *istioCVEManager) updateCVEs(newCVEs []*schema.NVDCVEFeedJSON10DefCVEItem) error {
+func (m *istioCVEManager) updateCVEs(newCVEs []*schema.NVDCVEFeedJSON10DefCVEItem) (retErr error) {
+	defer func() {
```

Comment:
Why not simply wrap a call to `updateCVEs` in `safe.RunE` from `github.com/stackrox/rox/pkg/safe`?